src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -176,9 +176,10 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
 
   const onChangeCutValue = (id: string, value: unknown) => {
     if (record && typeof value === 'string') {
-      const newTotal = calculatNewTotal(id, value);
-      setRecord({ ...record, [id]: value, seedsTested: newTotal || 0 });
-      setTotalSeedsTested(newTotal || 0);
+      const calculatedTotal = calculateNewTotal(id, value) || 0;
```

Comment:
probably dont' need the '|| 0' here

Reply:
Tried, but isNaN doesn't allow undefined

Reply:
ugh ok.. thanks for following up